### DISCUSSION TITLE
Problem Running the Python Examples

### DISCUSSION THREAD

**Discussion by Ahmad-197** [cubert-hyperspectral/cuvis.sdk#59] [Q&A] [answered]
I have followed the below steps to setup Cuvis for python
![image](https://github.com/cubert-hyperspectral/cuvis.sdk/assets/86935871/482e0d5e-7299-424e-84e8-85115c894e7f)

My Camera is connected and it is working fine. However when I try to run one of the examples from cuvis.python.examples I am receiving the following error:
![image](https://github.com/cubert-hyperspectral/cuvis.sdk/assets/86935871/6a9cdf5f-7ec3-4cd5-8e65-998f74a9ffd0)

Although I have had installed the Cuvis:
![image](https://github.com/cubert-hyperspectral/cuvis.sdk/assets/86935871/675c8790-727c-4e47-bf82-1ae066b0c8ed)

I am trying it on Windows Laptop and the Cuvis Ultis Touch version is 2.9.1 and SDK installer version is 3.2.3.

I am new to this, can someone tell me what I am missing?
Thankss!

**Comment by birkholz-cubert** [comment, marked as answer]
Hi,

i think you need to run it with `python EX05_recordSingleImages.py`.
Alternatively can you check with `which python` and `which python3` where the two python distributions are installed?

If that does not work let me know.
